.github/workflows/pkgci.yml

```diff
@@ -52,3 +52,9 @@ jobs:
     needs: [setup, build_packages]
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'regression_test_amdgpu_rocm')
     uses: ./.github/workflows/pkgci_regression_test_amdgpu_rocm.yml
+
+  test_tensorflow_cpu:
```

Comment:
If this job fails, will it block the merge?

Reply:
I don't believe so. Need to figure out how we want to handle that as more jobs move to pkgci. The AMDGPU jobs are currently just spinning waiting for runners...

For TF tests in this PR, I kind of want to put them at the same support status as the other pkgci regression tests (advisory/optional vs required).